src/model/UtilityBill.cpp

```diff
@@ -594,14 +594,14 @@ namespace detail {
         result.push_back("kBtu");
         result.push_back("MBtu");
         break;
-      //case FuelType::Coal:
+      case FuelType::Coal:
       //  result.push_back("tons");
       //  result.push_back("lbs");
       //  result.push_back("kLbs");
       //  result.push_back("MLbs");
-      //  result.push_back("kBtu");
-      //  result.push_back("MBtu");
-      //  break;
+        result.push_back("kBtu");
+        result.push_back("MBtu");
+        break;
```

Comment:
In UtilityBill, support Coal. But do not add weight-absed units as conversion isn't fixed, it depends on the type of coal.